### ISSUE TITLE
Causing Problems when Clicking Links in Gmail Messages?

### ISSUE THREAD

**Issue by WoofGrrrr** [piroor/newtabfromlocationbar#21]
I have FF Quantum (57.0.2) set to open links in new tabs.

After installing this "New Tab From Location Bar" extension, when I click a link in an Gmail message, I get TWO tabs.  One is blank, and the other displays the page I was expecting.

The first tab is blank, and it has a URL like this:  https://mail.google.com/_/scs/mail-static/_/js/..........

The second tab displays the page I was actually expecting.  When I rolled my cursor over the link in the Gmail message, the URL it shows is the one for this tab.

When I disable this extension, things go back to normal.  Only the expected tab opens.

Also note: If I switch this extension's "Blocking Method" option from "webRequest" to "webNavigation", the problem disappears.  When I switch back, it re-appears.

**Comment by ElhemEnohpi** [comment]
When you say "I have FF Quantum (57.0.2) set to open links in new tabs", do you mean the Firefox preference "Open new windows in a new tab instead", or something else?

**Comment by WoofGrrrr** [comment]
Yes, the "Open new windows in a new tab instead" option is enabled, plus these options in about:config

- browser.tabs.loadBookmarksInTabs
- browser.search.openintab

I also use extension "Open Link with New Tab".  (I did try disabling this extension, but it did not change anything.)

And as I said above:
  | Also note: If I switch this extension's "Blocking Method" option from "webRequest" to "webNavigation",
  | the problem disappears. When I switch back, it re-appears.



THis problem is also being discussed in the Mozilla Firefox Support forums.  It seems there are problems as well with links in Google News and Google+

**Comment by wjwaldron** [comment]
I'm experiencing the same problem. In my case, if I change method to webNavigation, I no longer get the extra tab from clicking a Gmail link, but new tabs are no longer opened from the location bar.

I have found that reverting from version 3.11 to 3.02 works for me (no extra tabs from Gmail links, location bar opens new tabs).

**Comment by WoofGrrrr** [comment]
That's strange.  It works okay for me when "webNavigation" is selected.  I am using version 3.1.1.

**Comment by wjwaldron** [comment]
Huh. It definitely wasn't for me previously. After reading your message, however, I updated back to 3.11 and chose webNavigation, and it now does indeed open location bar entries in new tabs.

**Comment by piroor** [comment]
A future version of Firefox may include a secret option for this purpose. We should wait to the bug https://bugzilla.mozilla.org/show_bug.cgi?id=1394304 become "fixed"...
